modules/etcd-increase-db.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="etcd-increase-db_{context}"]
+= Increasing the database size for etcd
+
+You can set the disk quota in gigabytes for each etcd instance. If you set a disk quota for your etcd instance, the instance can increase up to the value that you specify.
+
+You can specify integer values from 8 to 32. The default value is 8. You can specify only increasing values.
+
+:FeatureName: Increasing the database size for etcd
+include::snippets/technology-preview.adoc[]
+
+[id="etcd-change-db-size_{context}"]
+== Changing the etcd database size
+
+To change the database size for etcd, complete the following steps.
+
+.Procedure
+
+. Check to see what the current value is by entering the following command:
++
+[source,terminal]
+----
+$ oc describe etcd/cluster | grep "Backend Quota"
+----
++
+.Example output
+[source,terminal]
+----
+Backend Quota Gi B: <value>
+----
+
+. Change the value by entering the following command:
++
+[source,terminal]
+----
+oc patch etcd/cluster --type=merge -p '{"spec": {"backendQuotaGiB": <value>}}'
+----
+
+. Review the output:
++
+.Example output
+[source,terminal]
+----
+etcd.operator.openshift.io/cluster patched
```

Comment:
@dusk125 Just want to double check: Is this the correct output? I'd expect to see a value instead of a "cluster patched" message.

Reply:
Will verify this today after my cluster starts up... but if memory serves, that should be correct.

Reply:
Yeah there's no additional output other than the "patched" output.